src/UNL/Officefinder/Department.php

```diff
@@ -407,6 +423,15 @@ protected function getPublicProperties()
     function jsonSerialize()
     {
         $data = $this->getPublicProperties();
+
+        // Append Business Center Info if available
+        if ($this->hasBusinessCenterDepartment()) {
+            $businessCenter = $this->getBusinessCenterDepartment();
+            if ($businessCenter) {
+                $data['business_center'] = $businessCenter->getPublicProperties();
```

Comment:
IDK how I feel about attaching the full business_center here. Probably makes sense and is easier than requiring a separate request to fetch it using the bc_org_unit number if you want to work with it.